### PR TITLE
Adapt censoring (EXPOSUREAPP-7080)

### PR DESCRIPTION
**Pull request by kolyaopahle**
### Description
The CWA initially censored less information in logging output in tester builds to help with debugging.
We since discovered that the difference in logging is more irritating to our testers than it is helpful so this PR removes the feature.

### Steps to reproduce
1. Scan a QR code or interact with the app in a way that would result in a regtoken / qr code being logged
2. check if only the last 4 characters of the guid were logged

### CODE REVIEW

**Comment by LukasLechnerDev** [comment]
We are also doing things differently in `RatQrCodeCensor` => (`takeLast(28)` vs `takeLast(4)`)

**Comment by kolyaopahle** [comment]
> We are also doing things differently in `RatQrCodeCensor` => (`takeLast(28)` vs `takeLast(4)`)

i the rat censor is not merged into 2.3.x yet, or is it?

**Comment by LukasLechnerDev** [comment]
> > We are also doing things differently in `RatQrCodeCensor` => (`takeLast(28)` vs `takeLast(4)`)
> 
> i think the rat censor is not merged into 2.3.x yet, or is it?

At the time I wrote it, it wasn't. But since yesterday evening it is :).

**Comment by kolyaopahle** [comment]
> > > We are also doing things differently in `RatQrCodeCensor` => (`takeLast(28)` vs `takeLast(4)`)
> > 
> > 
> > i think the rat censor is not merged into 2.3.x yet, or is it?
> 
> At the time I wrote it, it wasn't. But since yesterday evening it is :).

should be done now :)

**Review by d4rken — DISMISSED** [review]
lgtm,

make CI happy though ;)

edit2: Seems to be a flaky test, i restarted it.

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/bugreporting/censors/RatQrCodeCensorTest.kt [hunk 1]
```diff
@@ -56,11 +56,6 @@ internal class RatQrCodeCensorTest {
         censor.checkLog(logLineToCensor) shouldBe logLineToCensor.copy(
             message = "Here comes the hash: SHA256HASH-ENDING-WITH-15ad of the rat test of RATest/FirstName RATest/LastName. He was born on RATest/DateOfBirth"
         )
-
-        every { CWADebug.isDeviceForTestersBuild } returns true
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/bugreporting/censors/RatQrCodeCensorTest.kt:60]: Can be removed here too: 

```
mockkObject(CWADebug)
every { CWADebug.isDeviceForTestersBuild } returns false
```

**Review by LukasLechnerDev — DISMISSED** [review]
LGTM! Small Cleanup would be nice. 👇

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/bugreporting/censors/RatQrCodeCensorTest.kt (hunk 1 above)
Comment (kolyaopahle) [reply Corona-Warn-App/src/test/java/de/rki/coronawarnapp/bugreporting/censors/RatQrCodeCensorTest.kt:60]: Ahhh i thought i got all of them 😄 should be gone now 👍

**Comment by kolyaopahle** [comment]
also noticed something weird in the coverage reporting by sonarcloud, coverage of CoronaTestCensor should be 100% according to intellij but sonarcloud reports 0%
image

**Review by LukasLechnerDev — APPROVED** [review]

**Review by d4rken — APPROVED** [review]

**Pull request merged**